django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -89,7 +89,7 @@
     color: #333;
 }
 
-#searchbar {
+#toolbar #searchbar {
```

Comment:
IMO we should add `#toolbar` also to the `#searchbar:focus` because it's ignored after this change.

Reply:
Yes, fair enough, I've added that. In my browser it's more or less invisible though, the browser's own focus styles more or less drown this out.